src/zjs_web_sockets.c

```diff
@@ -173,6 +170,21 @@ static void handle_error_arg(void *unused, jerry_value_t argv[], u32_t *argc,
     *argc = 1;
 }
 
+static void emit_error(jerry_value_t obj, const char *format, ...)
```

Comment:
Doesn't need to be done now, but would this be something we should put in the utils file?  Would it be useful for other modules?  Just a though.